queries/koto/highlights.scm

```diff
@@ -0,0 +1,131 @@
+[
+  "="
+  "+"
+  "-"
+  "*"
+  "/"
+  "%"
+  "+="
+  "-="
+  "*="
+  "/="
+  "%="
+  "=="
+  "!="
+  "<"
+  ">"
+  "<="
+  ">="
+  ".."
+  "..="
+] @operator
+
+[
+  "and"
+  "not"
+  "or"
+] @keyword.operator
+
+[
+  "return"
+  "yield"
+] @keyword.return
```

Comment:
```suggestion
"return" @keyword.return

"yield" @keyword.coroutine
```

Reply:
The docs in CONTRIBUTING.md say:

```
@keyword.return            ; keywords like `return` and `yield`
```

I don't have a strong opinion here, happy to go either way.

Reply:
yield depending on the language can be for return value of iterator or of coroutine. So which one is this `yield` meaning?

Reply:
Currently it's only for iterator output, but I think the coroutine description is appropriate so let's go with your suggestion. Thanks for the review!

Reply:
Oh yeah for iterators `@keyword.return` is fine.